docs/content/en/docs/troubleshooting/troubleshooting.md

```diff
@@ -244,6 +244,8 @@ Once the old KinD bootstrap cluster is deleted, you can rerun the `eksctl anywhe
 
 ### Cluster upgrade fails with management components on bootstrap cluster
 
+>**_NOTE:_**  KinD cluster is no longer used during upgrade of management cluster from v0.18.0 onwards.
```

Comment:
I'm not sure if it's enough only adding one note here. folks might miss it when just searching for a particular issue. Wonder if note should be in any reference to a kind cluster.

@jiayiwang7 opinions?

Reply:
I'd say the least we can do is to make this statement obvious enough with
```
{{% alert title="Important" color="warning" %}}

KinD cluster is no longer used during upgrade of management cluster from v0.18.0 onwards.

{{% /alert %}}
```

Then add something like

For EKS-A version v0.17.x and below, do xxx